subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractSmokeTest.groovy

```diff
@@ -120,7 +120,7 @@ abstract class AbstractSmokeTest extends Specification {
         static gradleVersions = "0.42.0"
 
         // https://plugins.gradle.org/plugin/org.gradle.playframework
-        static playframework = "0.12"
+        static playframework = "0.13"
```

Comment:
Do we also need to update springBoot's version in this file too? It looks like we're testing with 2.7.3, but maybe we're not trying to publish anything?

Reply:
Updated.  This removed an expected failure due to corrected input annotations.